install/0000_30_machine-api-operator_04_metal3provisioning.crd.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: metal3provisionings.metal3.io
+spec:
+  group: metal3.io
+  names:
+    kind: Metal3Provisioning
+    listKind: Metal3ProvisioningList
+    plural: metal3provisionings
+    shortNames:
+    - m3p
+    - m3ps
+    singular: metal3provisioning
+  scope: ""
+  subresources:
+    scale:
+      labelSelectorPath: .status.labelSelector
+      specReplicasPath: .spec.replicas
```

Comment:
Again, not a blocker for your own API, but we haven't generally placed scaling options here for operator-y things.  I just encourage you to be sure you want to.

Reply:
Not sure we want them so removing the scaling options.